chacha20poly1305/Cargo.toml

```diff
@@ -4,10 +4,16 @@ version = "0.0.0"
 authors = ["RustCrypto Developers"]
 edition = "2018"
 license = "MIT OR Apache-2.0"
-description = "CTR block mode of operation"
+description = """
+ChaCha20Poly1305 Authenticated Encryption with Additional Data Algorithm (RFC 8439)
+"""
 documentation = "chacha20poly1305"
 repository = "https://github.com/RustCrypto/AEADs"
 keywords = ["crypto", "cipher", "aead"]
 categories = ["cryptography", "no-std"]
 
 [dependencies]
+aead = "0.1"
+chacha20 = { version = "0.2.1", features = ["zeroize"] }
+poly1305 = "0.2"
+zeroize = { version = "0.9", default-features = false }
```

Comment:
I thought in other crates `zeroize` was optional?

Reply:
Happy to keep it that way if you want. The other crates have a lower MSRV than `zeroize`, so it had to be optional there. Could always be optional but on-by-default as well.

Reply:
Hm, I guess making it optional and enabling it by default will not make substantial difference compared to the non-optional approach, so we can leave it as-is.

BTW I wonder how zeroize will work in case like this:
```rust
struct Foo { .. }

struct Bar { foo: Foo, baz: Baz }
```

if both `Foo` and `Bar` implement `Zeroize`. Will it zeroize `Foo` space twice?

Reply:
@newpavlov I've been using explicit `Drop` handlers, and generally trying to push those down to the "leaf" data owner. If there are two `Drop` handlers like in your example, they will zero `foo` twice. I'd suggest only `Foo` impl `Drop` in the example above.